test/testdata/schema/ghc92/vscode-extension-schema.golden.json

```diff
@@ -249,6 +249,24 @@
         "scope": "resource",
         "type": "boolean"
     },
+    "haskell.plugin.semanticTokens.config.tokenMapping": {
+        "default": {
+            "class": "class",
+            "classMethod": "method",
+            "dataCon": "enumMember",
+            "function": "function",
+            "patternSyn": "macro",
+            "recField": "property",
+            "typeCon": "enum",
+            "typeFamily": "interface",
+            "typeSyn": "type",
+            "typeVariable": "typeParameter",
+            "variable": "variable"
+        },
+        "markdownDescription": "Configuration of the map from hs semantic token type to LSP default token type",
+        "scope": "resource",
+        "type": "object"
```

Comment:
This isn't very discoverable, I think it's not going to be clear to people how to edit this in the UI. Could we instead have an option per mapping? Then we could have an enum type for the token types as the option type, too?

Reply:
So more like 
```
haskell.plugin.semanticTokens.config.tokenMapping.class": {
        "default": "class",
        "markdownDescription": "LSP semantic token type to use for typeclass names",
        "type": "string",
        "enum": [ "class", "interface", ... ]
        ...
```

Then a) we get a specific description (so we can say exactly what the token type corresponds to, "class" is suggestive but better to be explicit), and people will get a dropdown menu of options for the token type.

I don't know how easy this is with our config mechanism, though.

Reply:
Customization might be needed, I'll take a look at the relevant code.

Reply:
done